kf/nfconfig_test.go

```diff
@@ -37,7 +38,7 @@ var (
 func setupDBTest() {
 	machineHostname, _ = os.Hostname()
 	hostInterfaces = make(map[string]bool)
-	hostInterfaces["enp0s3"] = true
+	hostInterfaces["fackif0"] = true
```

Comment:
```suggestion
	hostInterfaces["fakeif0"] = true
```
same throughout file